src/swagger/openapi.json

```diff
@@ -159,12 +170,40 @@
         ],
         "summary": "Create an article",
         "operationId": "createArticle",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "activeUserId",
```

Comment:
This should be `authentication-header`. See my other comments about `activeUserId` and `token`.